rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1613,6 +1580,14 @@ String loginWithCredential(@MultipartForm LoginForm multipart)
 	List<String> userspaceURIs(@HeaderParam("sessionid") String sessionId)
 			throws NotConnectedRestException, PermissionRestException;
 
+
+    @GET
+    @Path("globalspace")
+    @Produces("application/json")
+    List<String> globalspaceURIs(@HeaderParam("sessionid") String sessionId)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)